backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -61,6 +61,8 @@ case class DefaultStandardAsyncExecutionActorParams
   override val minimumRuntimeSettings: MinimumRuntimeSettings
 ) extends StandardAsyncExecutionActorParams
 
+case class ScriptPreambleData(bashString: String, executeInSubshell: Boolean)
```

Comment:
Can we set a default `executeInSubshell = true` here? That avoids us needing to set it in the PAPI backend and communicates to future developers that it's the safe "normal" choice.